src/main/java/jenkins/plugins/logstash/LogstashNotifier.java

```diff
@@ -56,11 +56,13 @@
 
   public int maxLines;
   public boolean failBuild;
+  public String customData;
 
   @DataBoundConstructor
-  public LogstashNotifier(int maxLines, boolean failBuild) {
+  public LogstashNotifier(int maxLines, boolean failBuild, String customData) {
```

Comment:
this will make `customData` a mandatory argument
this is a no-no as it will break existing usage
please use `@DataBoundSetter` instead